src/App.jsx

```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-export const tabs = [
+import { Tabs } from './components/Tabs';
+
+export const tabsFromServer = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+export const App = () => {
+  const [selectedTab, setSelectedTab] = useState(tabsFromServer[0]);
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
+  return (
+    <div className="section">
+      <h1 className="title">{`Selected tab is ${selectedTab.title}`}</h1>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <Tabs
+            tabs={tabsFromServer}
+            activeTab={selectedTab}
+            setActiveTab={setSelectedTab}
+          />
+        </div>
+        <div className="block" data-cy="TabContent">
+          {selectedTab.content}
+        </div>
```

Comment:
Move this logic to the Tabs component for pass tests

Reply:
follow current murkup this block is located outside of the tabs